CMakeLists.txt

```diff
@@ -78,7 +78,22 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/tools")
 
 # OpenMP
 find_package(OpenMP)
-if(OpenMP_CXX_FOUND OR OPENMP_FOUND) # OPENMP_FOUND for CMake <= 3.8
+# If OpenMP wasn't found, see if we can find it in the default Homebrew location
+if((NOT OpenMP_CXX_FOUND) AND EXISTS "/usr/local/opt/libomp/lib/libomp.dylib")
```

Comment:
I've done most development for DIPlib on a Mac, and never ran into this issue before. I'm honestly not really keen on putting a non-standard search location into the build scripts for the one use case.

From `/usr/local/share/cmake/Modules/FindOpenMP.cmake`:

>For some compilers, it may be necessary to add a header search path to find the relevant OpenMP headers.  This location may be language-specific.  Where this is needed, the module may attempt to find the location, but it can be provided directly by setting the `OpenMP_<lang>_INCLUDE_DIR` cache variable. Note that this variable is an _input_ control to the module.  Project code should use the `OpenMP_<lang>_INCLUDE_DIRS` _output_ variable if it needs to know what include directories are needed.

So why not just do `cmake -DOpenMP_CXX_INCLUDE_DIR=/usr/local/opt/libomp/include`?

Reply:
This is mostly to make it easier for other people to get up and running when using macos - this is quite an old patch, and maybe it's not needed anymore. if you don't like it, I'm fine with keeping it local to our own branch.